### PR TITLE
master-qa-72444

### PR DESCRIPTION
**Pull request by jaydawu**
https://issues.liferay.com/browse/LRQA-72444
Submit for master.

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase [hunk 1]
```diff
@@ -28,6 +28,84 @@ definition {
...
+
+		Click(
+			key_fieldFieldLabel = "English",
+			locator1 = "DDMEditStructure#FORM_FIELD_CONTAINER_LABEL");
+
+		Navigator.gotoNavTab(navTab = "Advanced");
+
+		Uncheck.uncheckToggleSwitch(
+			key_fieldName = "repeatable",
+			locator1 = "Checkbox#SWITCH");
```
Comment (brianwulbern) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase:52]: @Jayda-4145 there might be an existing macro for these steps - let's try to make sure individual functions are placed in macros, then we can use them later on.  This time it's ok.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase [hunk 2]
```diff
@@ -28,6 +28,84 @@ definition {
...
+			configurationName = "Language",
+			configurationScope = "Virtual Instance Scope");
+
+		PortalSettings.configureCurrentLanguagesCP(defaultPortalLanguage = "German (Germany)");
+
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+		WebContentNavigator.gotoAddWithStructureCP(structureName = "WC Structure Name");
+
+		AssertVisible(locator1 = "WCEditWebContent#SIDEBAR");
```
Comment (brianwulbern) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase:88]: @Jayda-4145 same idea here - push functions into macros

**Comment by brianwulbern** [comment]
No test changes needed :)

**Comment by brianwulbern** [comment]
ci:forward

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase (hunk 1 above)
Comment (jaydawu) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase:52]: @brianwulbern ok, got it! Thanks. I will try next time.

**Pull request closed without merging**